internal/pkgmanager/ecosystem.go

```diff
@@ -9,6 +10,8 @@ import (
 	"github.com/ossf/package-analysis/pkg/api/pkgecosystem"
 )
 
+var ErrNoArchiveURL = errors.New("no archive url found")
```

Comment:
nit: `"archive URL not found"` sounds a bit more natural

Reply:
fixed